.idea/.idea.mopub-android-xamarin/.idea/contentModel.xml

```diff
@@ -0,0 +1,389 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I'm guessing `.idea/` probably shouldn't be checked it; I recommend adding it to .gitignore.